tests/test_logs/test_server.py

```diff
@@ -0,0 +1,13 @@
+"""Test different server responses."""
+import sure  # noqa # pylint: disable=unused-import
+
+import moto.server as server
+
+
+def test_logs_list():
```

Comment:
The scaffolding-script creates this automatically, but it's not required.

It can be useful if you want to have fine-grained control over the HTTP request/response, but if everything you're testing is covered by the unit test already, there's no point in adding it.